tests/js/server/aql/aql-queries-geo.js

```diff
@@ -168,10 +164,6 @@ function ahuacatlLegacyGeoTestSuite () {
       actual = runQuery("FOR x IN NEAR(" + locations.name() + ", -70, 70, 2) SORT x.latitude, x.longitude DESC LIMIT 3 RETURN x");
       assertEqual(expected, actual);
     },
-    
```

Comment:
Why was this test removed?

Reply:
Because

> Fix S2LatLng -> S2Point works only for valid S2LatLng:
It was really strange. In some cases we produce error to user. In other ignore and works incorrect. In third makes normalize and works correct.
I think always produce error is better, but in such case we need way to update old user data. It's impossible now, so I think better to always normalize
So I change behavior: In some queries user couldn't specify not normalized points. It was error/invalid behavior, now they can and it will be valid

Reply:
Sorry, I do not get it. As far as I see, the `ToPoint` method of `S2LatLng` works for arbitrary latitude/longitute pairs, since it only applies `sin` and `cos` functions to the angles in radians. Therefore, the S2LatLng values do not have to be normalized at all for this to work.
Where in the PR is any change which changes behaviour which would merit removing the tests?
What are these cases in which we would produce an error vs. ignoring things?